otelcollector/prometheusreceiver/go.mod

```diff
@@ -1,49 +1,51 @@
 module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver
 
-go 1.22.0
+go 1.23.0
 
 require (
 	github.com/go-kit/log v0.2.1
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/snappy v0.0.4
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.117.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.117.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil v0.117.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheus v0.117.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.121.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.121.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil v0.121.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheus v0.121.0
 	github.com/prometheus/client_golang v1.20.5
-	github.com/prometheus/common v0.61.0
-	github.com/prometheus/prometheus v0.54.1
+	github.com/prometheus/common v0.62.0
+	github.com/prometheus/prometheus v0.300.1
```

Comment:
Thanks Kaveesh, looks like the UI is working with the latest commit now.

One more thing I thought of is the Prometheus UI testkube tests need to also have the update to prom 3.0 in the go.mod file of: /otelcollector/test/ginkgo-e2e/prometheusui/go.mod

Reply:
done, resolving